src/Compilers/CSharp/Test/CommandLine/SarifV1ErrorLoggerTests.cs

```diff
@@ -1,7 +1,12 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
+using System.Globalization;
```

Comment:
Are these using necessary?

Reply:
Nope, removing.

Reply:
Are you removing or just normalizing the change ...